niceman/utils.py

```diff
@@ -1193,4 +1194,13 @@ def _walk_up(path):
             yield path
             path = os.path.dirname(path)
 
+
+def is_subpath(path, directory):
+    """Test whether `path` is below (or is itself) `directory`.
+
+    Symbolic links are not resolved before the check.
+    """
+    return not os.path.relpath(path, directory).startswith(os.path.pardir)
```

Comment:
Out of curiosity, why not:
`return os.path.normpath(directory).startswith(os.path.normpath(path))
`
?

Reply:
Wow... mine would fail if the norm path doesn't end with slash because

 /a/foo.txt
 /a/foo.txt2

would be considered subdirectories.